### PR TITLE
Present producing stream with array as source

### PR DESCRIPTION
**Pull request by magzaba**
I am presenting multiple ways of creating a stream out of an array for reference and primitive types.

### CODE REVIEW

**Comment by marcin-ogorzalek-EPAM** [comment]
Please bump up the project's version

File: src/main/java/com/epam/prejap/ess/helpers/streamfromarray/StreamFromArray.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+import java.util.stream.DoubleStream;
+import java.util.stream.IntStream;
+import java.util.stream.LongStream;
+import java.util.stream.Stream;
+
+/**
+ * Presents different methods of producing a stream out of an array
+ * @author Magdalena Żaba
+ */
+public class StreamFromArray {
```
Comment (marcin-ogorzalek-EPAM) [inline src/main/java/com/epam/prejap/ess/helpers/streamfromarray/StreamFromArray.java:13]: Why `public` class?

Comment (magzaba) [reply src/main/java/com/epam/prejap/ess/helpers/streamfromarray/StreamFromArray.java]: changed to package private

**Comment by magzaba** [comment]
@marcin-ogorzalek-EPAM  I've pulled, merged and bumped up version

File: src/test/resources/testng.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
```
Comment (marcin-ogorzalek-EPAM) [inline src/test/resources/testng.xml:1]: I would like to ask you to add all aready present test's classes to this file. Tehn it would be ready to go.

**Comment by magzaba** [comment]
@marcin-ogorzalek-EPAM  test classes added

File: src/test/resources/testng.xml (hunk 2 above)
Comment (magzaba) [reply src/test/resources/testng.xml:1]: @marcin-ogorzalek-EPAM  test classes added

**Pull request merged**